tests-functional/tests/test_add_account.py

```diff
@@ -0,0 +1,148 @@
+import copy
+import pytest
+from resources.constants import new_account_data_1, new_account_data_2
+
+
+@pytest.mark.rpc
+class TestAddAccount:
+
+    @pytest.fixture(autouse=True)
+    def setup_backends(self, backend_new_profile):
+        self.account = backend_new_profile("sender")
+        self.account_data = copy.deepcopy(new_account_data_1)
+
+    def test_add_account_for_valid_key_uid(self):
+        self.account_data["key-uid"] = self.get_keypair_key_uid()
+        add_account_response = self.account.accounts_service.add_account(self.account_data)
+        self.account.verify_json_schema(add_account_response, method="accounts_addAccount")
+        assert "error" not in add_account_response
+
+        # After adding the account check that the get accounts will retrieve the new account
+        accounts_response = self.account.accounts_service.get_accounts()
+        accounts = accounts_response.get("result", [])
+        self.check_new_account_was_created(accounts, self.account_data)
+
+    def test_add_seconds_account_for_same_keypair(self):
+        key_uid = self.get_keypair_key_uid()
+
+        self.account_data["key-uid"] = key_uid
+        add_account_response = self.account.accounts_service.add_account(self.account_data)
+        assert "error" not in add_account_response
+
+        # Add a second account
+        self.account_data = new_account_data_2
```

Comment:
Does this create a deep copy?
(I don't know, just see that you `copy.deepcopy(new_account_data_1)` above)